pandas/io/json/normalize.py

```diff
@@ -224,6 +224,45 @@ def _pull_field(js, spec):
         sep = str(sep)
     meta_keys = [sep.join(val) for val in meta]
 
+    def _extract(obj, field, seen_meta, level):
+        """
+        Extract field from obj.
```

Comment:
Can you run this through `scripts/validate_docstrings.py` to see what it yields? I believe there should be a newline between the short / long summary here, though ideally the second line would be converted into a `Returns` section instead

Reply:
The 2nd line is not the long summary. @jreback requested the change to clarify the *return*.  I don't know the convention but I think this sentence shouldn't be in `Returns` because we don't `return` but updates these global variables. In any case, I made the sentence more concise to fit in 1 line.